src/modules/Hosts/Hosts/Views/MainPage.xaml

```diff
@@ -219,13 +189,14 @@
                             AutomationProperties.Name="{x:Bind Address, Mode=OneWay}"
                             Background="Transparent"
                             IsRightTapEnabled="True"
-                            RightTapped="Grid_RightTapped">
+                            RightTapped="Grid_RightTapped"
+                            ColumnSpacing="8">
                             <Grid.ColumnDefinitions>
-                                <ColumnDefinition Width="300" />
-                                <ColumnDefinition Width="*" />
-                                <ColumnDefinition Width="Auto" />
-                                <ColumnDefinition Width="Auto" />
-                                <ColumnDefinition Width="Auto" />
+                                <ColumnDefinition Width="256" x:Name="address"/>
+                                <ColumnDefinition Width="*" x:Name="comment"/>
+                                <ColumnDefinition Width="Auto" x:Name="status"/>
+                                <ColumnDefinition Width="Auto" x:Name="duplicate"/>
+                                <ColumnDefinition Width="Auto" x:Name="toggleswitch"/>
```

Comment:
If we don't need the names to reference it in XAML or code we shouldn't set them. Now it looks like something is happening with these columns.

Adding a `<!-- Address -->` comment is more appropriate.